<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CommonWidget</class>
 <widget class="QWidget" name="CommonWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>383</width>
    <height>410</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Options:</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QCheckBox" name="stopOnErrorCheckBox">
        <property name="font">
         <font>
          <pointsize>10</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Stop processing items in this extension if an
error occurs.</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="userContextCheckBox">
        <property name="font">
         <font>
          <pointsize>10</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Run in logged-on user's security
context (user policy option).</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="removeThisCheckBox">
        <property name="font">
         <font>
          <pointsize>10</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Remove this item when it is no longer applied.</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="applyOnceCheckBox">
        <property name="font">
         <font>
          <pointsize>10</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Apply once and do not reapply.</string>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QCheckBox" name="itemLevelCheckBox">
          <property name="font">
           <font>
            <pointsize>10</pointsize>
           </font>
          </property>
          <property name="text">
           <string>Item-level targeting.</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QToolButton" name="targetingToolButton">
          <property name="text">
           <string>Targetting...</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="descriptionLabel">
     <property name="text">
      <string>Description:</string>
     </property>
     <property name="buddy">
      <cstring>descriptionTextEdit</cstring>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QTextEdit" name="descriptionTextEdit"/>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
